src/main/java/com/mathworks/ci/BuildArtifactAction.java

```diff
@@ -222,6 +222,26 @@ private void iterateAllTaskAttributes(Entry pair, BuildArtifactData data) {
             case "skipped":
                 data.setTaskSkipped((Boolean) pair.getValue());
                 break;
+            case "skipReason":
+                String skipReasonKey = pair.getValue().toString();
+                String skipReason;
+                switch(skipReasonKey){
+                    case "UpToDate":
+                        skipReason = "up-to-date";
+                        break;
+                    case "UserSpecified":
+                    case "UserRequested":
+                        skipReason = "user requested";
+                        break;
+                    case "DependencyFailed":
+                        skipReason = "dependency failed";
```

Comment:
Do we use this anywhere else? Intuitively I think I would expect this to be something like "not run" rather than skipped. Definitely not a blocker for me though, because I think we have the freedom to change that later if we want.

Reply:
Yes, it definitely relates to "Not Run". I haven't seen it being used anywhere else. Just made it consistent with other reasons and if required we change in future.